### PR TITLE
Kill iOS components projects

### PR DESCRIPTION
**Pull request by eoger**
I need this to unblock the NSS work, and this is something we should have done a long time ago anyway.
Keep in mind that I'm doing the minimum possible work for this patch, we can probably factorize some stuff now (e.g. the Protobuf structs) now that we know it's a single .so file.

### CODE REVIEW

**Comment by thomcc** [comment]
We've talked about this before but concluded that we need to discuss this with the lockbox team first.

**Comment by thomcc** [comment]
@sashei @linuxwolf do y'all use the megazord (MozillaAppServices.framwork) on iOS or the separate FxAClient.framework/Logins.framework

**Comment by eoger** [comment]
Note that switching to the megazord is probably less than 30 minutes of work (well... from my experience doing the conversion on the sandvich sample app)

**Comment by garvankeeley** [comment]
Lockbox/Lockwise peers: feel free to ping me regarding trying the megazord in that iOS app, I can help with the setup there.

File: megazords/ios/MozillaAppServicesTests/PlacesTests.swift [hunk 1]
```diff
@@ -174,21 +174,21 @@ class PlacesTests: XCTestCase {
...
+    
     override func tearDown() {
         // This method is called after the invocation of each test method in the class.
     }
-
+    
     func testGetTree() {
         let db = api.getWriter()
-
+
```
Comment (thomcc) [inline megazords/ios/MozillaAppServicesTests/PlacesTests.swift]: plz no trailing whitespace

File: megazords/ios/MozillaAppServicesTests/PlacesTests.swift [hunk 2]
```diff
@@ -237,23 +237,23 @@ class PlacesTests: XCTestCase {
...
-
+        
         checkTree(try! db.getBookmark(guid: BookmarkRoots.MenuFolderGUID)!, [
             "guid": BookmarkRoots.MenuFolderGUID,
             "type": "folder",
             "childGUIDs": [newFolderGUID, sepGUID]
-        ], checkChildren: .onlyGUIDs)
-
-
+            ], checkChildren: .onlyGUIDs)
```
Comment (thomcc) [inline megazords/ios/MozillaAppServicesTests/PlacesTests.swift]: this indent makes things slightly worse.

**Comment by sashei** [comment]
@thomcc we do use the individual `Logins.framework` and `FxAClient.framework` libraries. 

I've filed an [issue in lockbox-ios](https://github.com/mozilla-lockbox/lockbox-ios/issues/907) for us to switch this over; I should have time to do this later in the week.

**Comment by eoger** [comment]
> Can we get rid of components/support/ios too?

Yes I'll file a follow-up for this and the rest after this lands.

**Comment by eoger** [comment]
I'm gonna go on and merge this, let us know if you need any assistance for the conversion Sasha.

**Pull request merged**